src/Block/Service/AbstractBlockService.php

```diff
@@ -35,10 +35,31 @@ abstract class AbstractBlockService implements BlockServiceInterface
      */
     private $twig;
 
-    public function __construct(string $name, Environment $twig)
+    /**
```

Comment:
These changes seem wrong.

Reply:
Why? We changed stuff in the stable and master branch:
**stable**: We deprecated the name
**master**: We added type hints

Removing the deprecated stuff should be done afterwards

Reply:
In that case, it's fine for me.